cockatrice/src/chatview/chatview.cpp

```diff
@@ -93,7 +93,7 @@ void ChatView::appendCardTag(QTextCursor &cursor, const QString &cardName)
     anchorFormat.setAnchor(true);
     anchorFormat.setAnchorHref("card://" + cardName);
     anchorFormat.setFontItalic(true);
-    
+
```

Comment:
If this were a big project with a dozen people working on it, such a change would be a problem:
 1. because it could introduce merge conflicts
 2. because the git log will store anyway the fact that you are the author that  last changed that line of code (think of git blame, etc..)

In our case I agree it's not important, you can leave it as it is.